documentation/modules/rn-27-resolved-issues.adoc

```diff
@@ -7,6 +7,26 @@
 
 {project-first} 2.7 has the following resolved issues:
 
+[id="resolved-issues-2-7-5_{context}"]
+== Resolved issues 2.7.5
+
+.XFS filesystem corruption after warm migration of VM from VMware
+
+In earlier releases of {project-short}, virtual machines (VM) were presenting XFS filesystem corruption after the warm migration from VMware to {virt} using {project-short}. This issue has been resolved in {project-short} 2.7.5. link:https://issues.redhat.com/browse/MTV-1679[(MTV-1679)]
+
+.Missing VM network-ID in the inventory
+
+In earlier releases of {project-short}, after creating a migration plan for VMs with a NSX-T network attached from vSphere, the VM network mapping was missing, and also adding network mapping could not list NSX-T networks as source networks. This issue has been resolved in {project-short} 2.7.5. link:https://issues.redhat.com/browse/MTV-1695[(MTV-1695)] and link:https://issues.redhat.com/browse/MTV-1140[(MTV-1140)]
```

Comment:
Am I ok to combine these two Jira tickets into one entry? 

I can see [MTV-1140](https://issues.redhat.com/browse/MTV-1140) is linked to [MTV-1695](https://issues.redhat.com/browse/MTV-1695)

![image](https://github.com/user-attachments/assets/3e06ecf4-8f66-41fb-b338-ca1c25091788)